decidim-core/db/migrate/20190412131728_fix_user_names.rb

```diff
@@ -15,7 +15,7 @@ def change
     characters_to_remove = "<>?%&^*\#@()[]=+:;\"{}\\|/"
 
     weird_characters.each do |character|
-      UserBaseEntity.where(deleted_at: nil).where("name like '%#{character}%' escape '\' OR nickname like '%#{character}%' escape '\'").find_each do |entity|
+      UserBaseEntity.where(deleted_at: nil).where("name like '%#{character}%' escape '' OR nickname like '%#{character}%' escape ''").find_each do |entity|
```

Comment:
This migration has been added in #5087, and it seems you have been doing the code review. 
By reading the [Postgres documentation](https://www.postgresql.org/docs/15/functions-matching.html#FUNCTIONS-LIKE)  i am going through: 
> It's also possible to select no escape character by writing ESCAPE ''. This effectively disables the escape mechanism, which makes it impossible to turn off the special meaning of underscore and percent signs in the pattern.
>
> According to the SQL standard, omitting ESCAPE means there is no escape character (rather than defaulting to a backslash), and a zero-length ESCAPE value is disallowed. PostgreSQL's behavior in this regard is therefore slightly nonstandard.

And i understand that we are using `escape` in order to disable the default escape character which is `\`. 
I am not feeling we have anything to work on this matter.

Reply:
@ahukkanen this is fixed by [7411533](https://github.com/decidim/decidim/pull/10932/commits/74115330cfb9b7ef88aad3c712fa000d7bd22e13)